app/controllers/houston/dashboards/staging_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Houston::Dashboards::StagingController < ApplicationController
   layout "houston/dashboards/dashboard"
-  STAGING_PROJECTS = %w(members unite ledger bsb)
+  STAGING_PROJECTS = %w(members unite ledger bsb members-dav)
```

Comment:
From what I can tell, `find_all` is no longer a method in ActiveRecord. I'm looking into finding another way to pass a block into the API or I may just create a sql query.